.github/workflows/expose_link.yml

```
@@ -11,12 +11,16 @@ jobs:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           artifact-path: 0/result.html
           circleci-jobs: build_docs
-
-  circleci_docs_artifacts_redirector_job:
+          job-title: benchmark result artifact
+---
+name: Expose docs artifact
+on: [status]
+jobs:
```

In the end, I don't think we actually need to have 2 workflows. Both jobs are running on master so I would say this is working. Note that we cannot see what is going on in this PR because it only changes when merge in master.
```suggestion
          job-title: benchmark result artifact
```